packages/next/src/utils.ts

```diff
@@ -3307,19 +3308,14 @@ export function isApiPage(page: string | undefined) {
     .match(/(serverless|server)\/pages\/api(\/|\.js$)/);
 }
 
-/** @deprecated */
-export type VariantsManifestLegacy = Record<
-  string,
-  {
-    defaultValue?: unknown;
-    metadata?: Record<string, unknown>;
-  }
->;
+export type VariantsManifest = {
```

Comment:
Should this be called `FlagsManifest`

Reply:
@dferber90 that will depend on what Next.js will ultimately call it. Kept it like this to have a differnce between Next and the rest for now